pages/1.10/installing/evaluation/mesosphere-supported-methods/gcp/index.md

```diff
@@ -70,298 +71,510 @@ us-west1
+    ```bash
+    terraform plan -out=plan.out
+    ```
 
-Now, you should see a message like the one below, confirming that you have successfully saved to the `plan.out` file. This file should appear in your `dcos-tf-gcp-demo` folder alongside `main.tf`.
+    Writing the execution plan to a file allows you to pass the execution plan to the `apply` command and guarantees the accuracy of the plan. Every time you run the `terraform plan` command, its output provides details about the resources the plan will add, change, or destroy. Since this sample installation creates the first DC/OS cluster, the output indicates that execution of the plan adds 38 pieces of infrastructure/resources.
 
-<p align=center>  
-<img src="./images/install/terraform-plan.png" />
-</p>
+    The plan output file is created in the `dcos-tf-gcp-demo` folder alongside the `main.tf` file. This file is **only readable** by Terraform, however.
 
-Every time you run `terraform plan`, the output will always detail the resources your plan will be adding, changing or destroying.  Since we are creating our DC/OS cluster for the very first time, our output tells us that our plan will result in adding 38 pieces of infrastructure/resources.
+    A message similar to the following confirms that you have successfully saved the execution plan to the `plan.out` file. 
+    
+  <p align=center>
+  <img src="./images/install/terraform-plan.png" />
+  </p>
 
-5) The next step is to get Terraform to build/deploy our plan.  Run the command below.
+6) Run the following command to deploy the plan that builds the cluster:
 
-```bash
-terraform apply plan.out
-```
+    ```bash
+    terraform apply plan.out
+    ```
 
-Once Terraform has completed applying our plan, you should see output similar to the following:  
+    Once Terraform has completed applying the plan, you should see output similar to the following:
 
-<p align=center>
-<img src="./images/install/terraform-apply.png" />
-</p>
+  <p align=center><img src="./images/install/terraform-apply.png" /></p>
 
-Congratulations - you’re done!  In just 4 steps, you’ve successfully installed a DC/OS cluster on GCP!
+### Congratulations! 
+You have successfully installed a DC/OS cluster on GCP with minimal configuration or customization. From here, you can log in to begin using the new cluster.
 
 <p align=center>
 <img src="./images/install/dcos-login.png" />
 </p>
 
+After you log in, the DC/OS dashboard is displayed.
+
 <p align=center>
 <img src="./images/install/dcos-ui.png" />
 </p>
 
-# Scaling Your Cluster
+For additional information about creating a cluster on GCP with more advanced configuration options, see [Advanced GCP cluster configuration options](#AdvancedGCP).
+
+# Scaling the cluster
 Terraform makes it easy to scale your cluster to add additional agents (public or private) once the initial cluster has been created. Use the instructions below.
```

Comment:
The Universal Installer makes it easy to scale your DC/OS cluster.  You can quickly add additional agent nodes, public or private, once the initial DC/OS cluster has been created with the instructions below.